probert/network.py

```diff
@@ -734,6 +743,9 @@ def route_change(self, action, data):
         self.receiver.route_change(action, data)
 
     def trigger_scan(self, ifindex):
+        if not self.wlan_listener:
```

Comment:
nit: inconsistencies between usage of `is not None` and `not`

Reply:
Thanks!